src/cljs/ataru/hakija/application_handlers.cljs

```diff
@@ -102,12 +102,13 @@
   (assoc-in db [:application :submit-status] :submitted))
 
 (defn send-application [db method]
-  {:db       (-> db (assoc-in [:application :submit-status] :submitting) (dissoc :error))
-   :http     {:method        method
-              :url           "/hakemus/api/application"
-              :post-data     (create-application-to-submit (:application db) (:form db) (get-in db [:form :selected-language]))
-              :handler       :application/handle-submit-response
-              :error-handler :application/handle-submit-error}})
+  (when-not (-> db :application :submit-status)
```

Comment:
Shouldn't we return a `{:db db}` or something from the handler function when `:submit-status` doesn't exist? If we return a `nil` then the whole UI application will crash because the application state is lost.

Reply:
`send-application` is called from a `event-fx` handler from which it is ok to return `nil`.